README.md

```diff
@@ -226,4 +228,42 @@ demo1-argocd-server                           LoadBalancer   10.96.0.88    10.22
 ```
 6. If you encounter a DockerHub rate-limiting for the Redis image, use a proxy-cache or host the image on another registry. The sample `argocd-instance.yaml` shows how to reference an alternate image location.  
 
-- For advanced configurations, check the [detailed reference](https://argocd-operator.readthedocs.io/en/latest/reference/argocd/) and [sample usage](https://argocd-operator.readthedocs.io/en/latest/usage/basics/)
\ No newline at end of file
+- For advanced configurations, check the [detailed reference](https://argocd-operator.readthedocs.io/en/latest/reference/argocd/) and [sample usage](https://argocd-operator.readthedocs.io/en/latest/usage/basics/)
+
+## External Secrets Operator
+
+<img src="supervisor-services-labs/external-secrets-operator/externalsecrets.png" width="200" title="External Secrets Logo" id="externalsecrets">
+
+External Secrets Operator is a Kubernetes operator that integrates external secret management systems like AWS Secrets Manager, HashiCorp Vault, Google Secrets Manager, Azure Key Vault, IBM Cloud Secrets Manager, CyberArk Conjur, etc. The operator reads information from external APIs and automatically injects the values into a Kubernetes Secret. For a detailed description of how to consume External Secrets Operator, visit [External Secrets Operator project](https://external-secrets.io/latest/)
+
+### External Secrets Operator Versions
+
+- Download latest version: [External Secrets Operator v0.9.14](supervisor-services-labs/external-secrets-operator/external-secrets-operator.yaml)
+
+External Secrets Operator Sample `values.yaml` - None
+
+- We do not provide any default `values.yaml` for this package. This operator requires minimal configurations, and the necessary pods get deployed in the `svc-external-secrets-operator-domain-xxx` namespace.
+
+Usage: 
+- Since the External Secrets Operator integrates with multiple providers, usage varies based on the types of secret stores accessed and secrets consumed. Once the External Secrets Operator has been deployed successfully on the Supervisor, basic operations include creating a SecretStore object and an ExternalSecret object within your vSphere Namespace.
```

Comment:
creating a SecretStore object and an ExternalSecret object -> creating a `SecretStore` object and an `ExternalSecret` object